src/app/testing/testing.component.html

```diff
@@ -9,7 +9,20 @@ <h1 class="page-header">Testing and Validation</h1>
+                <div class="field-item even" property="content:encoded">
+                  <h2>Continuous Integration Testing</h2>
+                  <p>EnergyPlus uses an extensive set of <a href=https://en.wikipedia.org/wiki/Continuous_integration>Continuous Integration</a> tests to ensure that software code fixes to identified issues, as well as new code adding additional features to EnergyPlus, are not causing unexpected changes in the results. This is part of the <a href=https://github.com/NREL/EnergyPlus/actions>GitHub workflows</a> that test the software for each new change. The types of tests include:</p>
+                  <ul>
+                    <li>Regression tests run hundreds of EnergyPlus building models and automatically compare the results of the simulations between the new and existing source code.</li>
+                    <li>Unit tests that exercise individual or small groups of functions in the source code to ensure that changes to small units of code do not negatively impact the overall results.</li>
+                    <li>Build tests that create and compile EnergyPlus and almost all supporting utilities on various platforms (Windows, MacOS, Linux), including documentation and package installers, to ensure that deployments of new versions of EnergyPlus are done smoothly.</li>
+                    <li>Miscellaneous tests exercise the multitude of interacting pieces of the entire EnergyPlus package to reduce the unforeseen impacts of changing components on the overall software.</li>
+                  </ul>
+                  <p>Tracking of the speed and the number of lines and functions that are covered by the tests is also included. Most of these tests can be performed locally by each developer on the team so they can provide the best changes prior to committing them to the common repository.</p>
+                  <h2>Current Validation Testing</h2>
+                  <p>An important ongoing part of EnergyPlus development is testing using industry-standard methods as major builds are completed. The goal is to make EnergyPlus as bug-free as possible. Currently, the validation primarily follows ASHRAE Standard 140 and is conducted as part of the <a href=https://openstudio.net>OpenStudio</a> validation using the same standard. The automated validation system can be seen <a href=https://github.com/NREL/BESTEST-GSR>on the BESTEST-GSR repository</a>.  Note that the results are posted under [individual releases]( https://github.com/NREL/BESTEST-GSR/releases) that generally correspond with software releases of OpenStudio and EnergyPlus.</p>
+                  <h2>Earlier Validation Testing</h2>
+                  <p>Historically, two categories of validation tests were conducted:</p>
```

Comment:
This is an update to the testing page to better reflect current testing.  I'm not set up right now to render the whole web site, but this does look OK when I just render it as raw html.